proto/types/action.proto

```diff
@@ -97,7 +97,7 @@ message StakeRestake {
 // move the bucket to vote for another candidate or transfer the ownership of bucket to another voters
 message StakeMove {
   uint64 bucketIndex = 1;
-  string candidateAddress = 2;
+  string candidateName = 2;
```

Comment:
just use "name" or "nameOrAddress"
in case of vote for another cand, it is the new candidate name
in case of transfer ownership, it is the io address of new voter